### PR TITLE
[Workspace] Hide first home breadcrumbs within workspace

### PR DESCRIPTION
**Pull request by Hailong-am**
### Description

Hide the home breadcrumbs when we are in a workspace, but keep it in recent popover

image

## Changelog

- feat: [Workspace] Hide home breadcrumbs when in a workspace

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by Hailong-am** [comment]
@AMoo-Miki @zhongnansu can you help to review this change?

File: src/core/public/chrome/ui/header/header_breadcrumbs.tsx [hunk 1]
```diff
@@ -73,6 +75,10 @@ export function HeaderBreadcrumbs({
     crumbs = breadcrumbEnricher(crumbs);
   }
 
+  if (dropHomeFromBreadcrumb && crumbs.length && Object.hasOwn(crumbs[0], 'home')) {
```
Comment (ruanyl) [inline src/core/public/chrome/ui/header/header_breadcrumbs.tsx:78]: Do we need to check if home is true?

Comment (Hailong-am) [reply src/core/public/chrome/ui/header/header_breadcrumbs.tsx:78]: we could add this condition as well. There only have one place to set this additional property, i think it's ok only check existence.

**Review by zhongnansu — APPROVED** [review]
lgtm

**Review by SuZhou-Joe — APPROVED** [review]

File: src/core/public/chrome/ui/header/header_breadcrumbs.tsx (hunk 1 above)
Comment (ruanyl) [reply src/core/public/chrome/ui/header/header_breadcrumbs.tsx:78]: Can we have the assumption that when `home` is set, it must be true?

**Review by ruanyl — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
`Run cypress tests (osd:ciGroup6)` should be irrelevant to the code changes.

**Pull request merged**